sale_view_adj_hls/views/sale_order_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">view.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='warehouse_id']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
```

Comment:
Delete this xpath entry as `warehouse_id` doesn't belong to this view.  Create the following record instead.
```
    <record id="view_order_form_inherit_sale_stock" model="ir.ui.view">
        <field name="name">view.order.form.inherit.sale.stock</field>
        <field name="model">sale.order</field>
        <field name="inherit_id" ref="sale_stock.view_order_form_inherit_sale_stock"/>
        <field name="arch" type="xml">
            <xpath expr="//field[@name='warehouse_id']" position="replace"/>
        </field>
    </record>
```